src/components/PositionsList/PositionsList.tsx

```diff
@@ -35,6 +36,28 @@ export const PositionsList: React.FC<IProp> = ({
   noConnectedBlockerProps
 }) => {
   const classes = useStyle()
+  const [page, setPage] = useState(1)
+  const itemsPerPage = 6
+  const handleChangePagination = (page: number): void => {
+    setPage(page)
+  }
+  function paginator(data: ILiquidityItem[], currentPage: number, perPageItems: number) {
```

Comment:
i think that arguments data and perPageItems are not necessary there, you can get this data from data prop and itemsPerPage variable